src/mpid/ch4/shm/ipc/gpu/gpu_post.c

```diff
@@ -85,6 +85,21 @@
         read - use read protocol
         write - use write protocol if remote device is visible
 
+    - name        : MPIR_CVAR_CH4_GPU_RMA_ENGINE_TYPE
+      category    : CH4
+      type        : enum
+      default     : auto
+      class       : none
+      verbosity   : MPI_T_VERBOSITY_USER_BASIC
+      scope       : MPI_T_SCOPE_ALL_EQ
+      description : |-
+        By default, select engine type automatically
+        yaksa - don't select, use yaksa
+        auto - select automatically
+        compute - use compute engine
+        copy_high_bandwidth - use high-bandwidth copy engine
+        copy_low_latency - use low-latency copy engine
+
```

Comment:
Please add commit message describing what's in the commit. For example, in needs describe that the commit is- 
* adding a CVAR for selecting GPU copy engine in the RMA path (is it ofi only or shm path? If latter, is the name too general?) 
* adding code for the yaksa bypass engine types.
* adding global_dev_id to shared_table (and why we need it)

Reply:
Added.

> is it ofi only or shm path?

Yes it is shm path only

Reply:
Then shouldn't we add `SHM` to the CVAR name?

Reply:
I think we left out `SHM` to match the other CVAR definitions, but still failed to follow the same naming convention.. What about changing to `MPIR_CVAR_CH4_IPC_GPU_RMA_ENGINE_TYPE` to match the others? Adding IPC I think clarifies it is intended for shm path while sticking with the existing naming convention

Reply:
Sounds good!

Reply:
Done